trivy.yaml

```diff
@@ -0,0 +1,28 @@
+timeout: 10m
+format: json
+dependency-tree: true
+list-all-pkgs: true
+exit-code: 1
+output: trivy-output.json
+# All severity levels
+severity:
+  - UNKNOWN
+  - LOW
+  - MEDIUM
+  - HIGH
+  - CRITICAL
+scan:
+  skip-dirs:
+    - .github/
+    - core/.terraform/
+    - shared/.terraform/
+
+  scanners:
+    - vuln
```

Comment:
does it have `vuln` scanner for terraform?

from this page https://aquasecurity.github.io/trivy/v0.47/docs/coverage/iac/terraform/

they have only misconfiguration and secret scanners

Reply:
I think they are missing somehow because they're having the section for that
https://aquasecurity.github.io/trivy/v0.47/docs/scanner/vulnerability/

The config file setup can be found here: https://aquasecurity.github.io/trivy/v0.47/docs/references/configuration/config-file/